.github/workflows/test-docs.yml

```diff
@@ -0,0 +1,29 @@
+name: Test Documentation Build
+
+on:
+  push:
+    paths:
+      - 'docs/**'
+  pull_request:
+    paths:
+      - 'docs/**'
+
+jobs:
+  test_docs:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
```

Comment:
I remain only 3.10 to match the same version in https://github.com/jointakahe/takahe/blob/main/.readthedocs.yaml#L7.